### PR TITLE
Implementation of #39 (Pt 2)

### PR DESCRIPTION
**Pull request by Waldleufer**
Distributing CI - The CI will break for this branch since there are unresolved eslint warnings.

Regarding: Feature #39

### CODE REVIEW

**Comment by Waldleufer** [comment]
Pt 1: #59 

Sorry for merging that PR prior but I wanted to get the CI Pipeline through ;)

**Review by SaiVarunVaranasi — COMMENTED** [review]
:-)

**Review by SaiVarunVaranasi — CHANGES_REQUESTED** [review]
Please check why the CI deployment is failing for the branch before merging with other branches. Other than that the code looks good.

**Review by SaiVarunVaranasi — COMMENTED** [review]
Can be merged if the issue is deemed complete

**Comment by SaiVarunVaranasi** [comment]
> Please check why the CI deployment is failing for the branch before merging with other branches. Other than that the code looks good.

I removed the additional "Axios import" in DetailsComponent.js and the build is now successful.

**Review by SaiVarunVaranasi — APPROVED** [review]

**Review by Waldleufer — COMMENTED** [review]
looks good now

File: frontend/src/components/details/PanelComponent.jsx
Comment (gandompm) [reply frontend/src/components/details/PanelComponent.jsx]: This will be improved during the component styling @Waldleufer

**Pull request merged**
